litestar/handlers/websocket_handlers/listener.py

```diff
@@ -281,8 +277,8 @@ def _validate_handler_function(self) -> None:
     async def default_connection_lifespan(
         self,
         socket: WebSocket,
-        on_accept_dependencies: Optional[Dict[str, Any]] = None,  # noqa: UP006, UP007
-        on_disconnect_dependencies: Optional[Dict[str, Any]] = None,  # noqa: UP006, UP007
+        on_accept_dependencies: Optional[dict[str, Any]] = None,  # noqa: UP007
```

Comment:
Should we remove `UP007` and the `Optional` as well?

Reply:
Sure because Python 3.8 also have support of `|` operator as `Union`, we can use `SomeType | None`.

Reply:
Done